lib/web_ui/lib/src/engine/configuration.dart

```diff
@@ -183,6 +185,32 @@ class FlutterConfiguration {
     defaultValue: 'https://unpkg.com/canvaskit-wasm@$_canvaskitVersion/bin/',
   );
 
+  /// The variant of CanvasKit to download.
+  ///
+  /// Available values are:
+  ///
+  /// * `auto` - the default value. The engine will automatically detect the
+  /// best variant to use based on the browser.
+  ///
+  /// * `full` - the full variant of CanvasKit that can be used in any browser.
+  ///
+  /// * `chromium` - the lite variant of CanvasKit that can be used in
+  /// Chromium-based browsers.
+  CanvasKitVariant get canvasKitVariant {
+    final String variant = _configuration?.canvasKitVariant ?? 'auto';
+    switch (variant) {
```

Comment:
Not a blocker, but instead of doing your own switch here, you can do `CanvasKitVariant.values.byName(variant)` here.

Reply:
Done